idaes/config.py

```diff
@@ -482,13 +482,16 @@ def create_dir(d):
 
 def get_data_directory():
     """Return the standard data directory for idaes, based on the OS."""
-    try:
-        if os.name == 'nt':  # Windows
-            data_directory = os.path.join(os.environ['LOCALAPPDATA'], "idaes")
-        else:  # any other OS
-            data_directory = os.path.join(os.environ['HOME'], ".idaes")
-    except AttributeError:
-        data_directory = None
+    if 'IDAES_DATA' in os.environ and os.environ['IDAES_DATA']:
```

Comment:
Might be cleaner to expand `os.environ['IDAES_DATA']` to `os.environ['IDAES_DATA'] is not None` to clarify what's going on here.

Reply:
I was actually checking for a blank string, but maybe I should check that up one level is a valid directory.  I'll make that change.